packages/zoltan2/src/algorithms/color/Zoltan2_AlgSerialGreedy.hpp

```diff
@@ -120,7 +121,7 @@ class AlgSerialGreedy : public Algorithm<Adapter>
   void colorCrsGraph(
     const size_t nVtx,
     ArrayView<const gno_t> edgeIds,
-    ArrayView<const lno_t> offsets,
+    ArrayView<const offset_t> offsets,
```

Comment:
This change of type causes a compiler warning at line 133:
Trilinos/packages/zoltan2/src/algorithms/color/Zoltan2_AlgSerialGreedy.hpp:133:7: warning: comparison between signed and unsigned integer expressions [-Wsign-compare]